pkg/sloop/server/internal/config/config.go

```diff
@@ -78,6 +87,15 @@ func registerFlags(fs *flag.FlagSet, config *SloopConfig) {
 	fs.StringVar(&config.ApiServerHost, "apiserver-host", "", "Kubernetes API server endpoint")
 	fs.BoolVar(&config.WatchCrds, "watch-crds", true, "Watch for activity for CRDs")
 	fs.StringVar(&config.RestoreDatabaseFile, "restore-database-file", "", "Restore database from backup file into current context.")
+	fs.Float64Var(&config.BadgerDiscardRatio, "badger-discard-ratio", 0.01, "Badger value log GC uses this value to decide if it wants to compact a vlog file.  Smaller values free more disk space but use more computing resources")
+	fs.DurationVar(&config.BadgerVLogGCFreq, "badger-vlog-gc-freq", time.Minute*5, "Frequency of running badger's ValueLogGC")
+	fs.BoolVar(&config.BadgerVLogGCLoop, "badger-vlog-gc-loop", true, "If set, rung badger ValueLogGC in a loop until it finds no data to free")
+	fs.Int64Var(&config.BadgerMaxTableSize, "badger-max-table-size", 0, "Max LSM table size in bytes")
```

Comment:
Would it help to add something to the description (and others) indicating that 0 implies not setting a value?

Reply:
added, thanks